src/main/java/de/tum/in/www1/artemis/domain/Complaint.java

```diff
@@ -25,8 +25,10 @@
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class Complaint extends DomainObject {
 
-    @Column(name = "complaint_text", length = 2000)
-    @Size(max = 2000)
+    public static final int COMPLAINT_TEXT_LIMIT = 5000;
```

Comment:
Maybe move this to `de.tum.in.www1.artemis.config.Constants`? 
That class also contains stuff like `FEEDBACK_DETAIL_TEXT_MAX_CHARACTERS`, so it should fit in there.

Reply:
Good to know. I didn't know there was a perfect place for constants :D Thank you!